### ISSUE TITLE
Add log statements

### ISSUE THREAD

**Issue by kevaundray** [noir-lang/noir#383]

**Comment by kevaundray** [comment]
@vezenovm doesn't #622 only solve the addition of basic strings?

**Comment by vezenovm** [comment]
Yes I forgot that putting this issue in that PR description would automatically close the issue. Reopening it until https://github.com/noir-lang/acvm/pull/61 and #630 are merged into master.
